src/server/saved_objects/serialization/index.ts

```diff
@@ -22,7 +22,7 @@
  * the raw document format as stored in ElasticSearch.
  */
 
-import uuid from 'uuid';
+import { v1 as uuidv1 } from 'uuid';
```

Comment:
lets add the the related errors you show in the failed jenkins job as evidence to support this change

Reply:
Error log added